website/docs/docs/dbt-versions/release-notes/08-May-2023/may-ide-updates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "May IDE updates and fixes"
+id: "may-ide-updates"
+description: "May 2023 release note: We've ."
+sidebar_label: "Update and fixes: IDE"
+sidebar_position: 2
+tags: [May-2023, IDE]
+---
+
+To continue improving your [Cloud IDE](https://docs.getdbt.com/docs/cloud/develop-in-the-cloud) development experience, the dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability ✨.
+
+Stay up-to-date with [IDE-related changes](https://docs.getdbt.com/tags/ide).
+
+## New features 
+- Lint via SQLFluff in beta (reach out if interested, otherwise this will be available for multi-tenant soon!)
```

Comment:
let's just say "Lint via SQL Fluff is now available in beta (GA over the next 2-3 weeks)" and remove the comment about reach out